src/Elastic.Apm/Config/AbstractConfigurationWithEnvFallbackReader.cs

```diff
@@ -84,7 +84,7 @@ internal AbstractConfigurationWithEnvFallbackReader(IApmLogger logger, string de
 			ParseMetricsInterval(Read(KeyNames.MetricsInterval, EnvVarNames.MetricsInterval));
 
 		public bool Recording =>
-			ParseEnabled(Read(KeyNames.Enabled, EnvVarNames.Enabled));
+			ParseEnabled(Read(KeyNames.Recording, EnvVarNames.Recording));
```

Comment:
@russcam 

Hi, russcam 
Do you mean `ParseRecording`?

Reply:
All right, we really need a test for this 🤣